src/qml/filters/deband/ui.qml

```diff
@@ -0,0 +1,440 @@
+            maximumValue: 64
+            decimals: 1
+            onValueChanged: filter.set(rangeParam, idRangeRand.checked ? rootToSqr(value) : -rootToSqr(value))
+        }
+        Shotcut.UndoButton {
+            onClicked: {
+                filter.set(rangeParam, rangeDefault)
+                setControls()
+            }
+        }
+
+        // Row split
+
+        Label {}
+        CheckBox {
+            id: idRangeRand
+            text: qsTr('Randomize between zero and value')
+            onClicked: filter.set(rangeParam, checked ? rootToSqr(idRange.value) : -rootToSqr(idRange.value))
+        }
+        Label {}
+
+        // Row split
+
+        Label {
+            text: qsTr('Direction')
+            Layout.alignment: Qt.AlignRight
+        }
+        Shotcut.SliderSpinner {
+            id: idDirection
+            minimumValue: 0
+            maximumValue: 360
+            suffix: ' °'
+            Shotcut.HoverTip { text: qsTr('Up = 270°\nDown = 90°\nLeft = 180°\nRight = 0°/360°') }
+            onValueChanged: filter.set(directionParam, idDirectionRand.checked ? degToRad(value) : -degToRad(value))
+        }
+        Shotcut.UndoButton {
+            onClicked: {
+                filter.set(directionParam, directionDefault)
+                setControls()
+            }
+        }
+
+        // Row split
+
+        Label {}
+        CheckBox {
+            id: idDirectionRand
+            text: qsTr('Randomize between 0° and value')
+            onClicked: filter.set(directionParam, checked ? degToRad(idDirection.value) : -degToRad(idDirection.value))
+        }
+        Label {}
+
+        // Filler
+
+        Label {
+            Layout.columnSpan: 3
+        }
+
+        // Row split
+
+        Label {}
+        CheckBox {
+            id: idBlur
+            text: qsTr('Use average of neighbors')
+            Shotcut.HoverTip { text: qsTr('Compare to thresholds using average versus exact neighbor values') }
+            onClicked: filter.set(blurParam, checked)
+        }
+        Shotcut.UndoButton {
+            onClicked: {
+                filter.set(blurParam, blurDefault)
+                idBlur.checked = blurDefault
+            }
+        }
+
+        // Row split
+
+        Label {}
+        CheckBox {
+            id: idCoupling
+            text: qsTr('All components required')
+            Shotcut.HoverTip { text: qsTr('Deband only if all pixel components (including alpha) are within thresholds') }
+            onClicked: filter.set(couplingParam, checked)
+        }
+        Shotcut.UndoButton {
+            onClicked: {
+                filter.set(couplingParam, couplingDefault)
+                idCoupling.checked = couplingDefault
+            }
+        }
+
+        // Filler
+
+        Label {
+            Layout.columnSpan: 3
+        }
+
+        // Row split
+
+        Label {
+            text: qsTr('More information') + ': <a href="http://ffmpeg.org/ffmpeg-all.html#deband">FFmpeg deband</a>'
```

Comment:
I appreciate what you are trying to do here. But we do not do this for other filters and I am not thrilled that it exposes some basic users to advanced things they may not understand. Can we extract the most important parts of the ffmpeg documentation and incorporate it into the tooltips? I could even support some paragraph text here if it would avoid the link to ffmpeg. I am open minded on this and mostly looking for brainstorm/discussion.

Reply:
I'm fine with removing the entire block.  There is plenty of documentation on the forum that should be more accessible to beginning users.

Reply:
Agreed. It would be great if you could also contribute a Documentation page on the forum for this filter.

Reply:
Documentation exists in the forum at https://forum.shotcut.org/t/introducing-a-new-filter-deband/28302

I will update references to labels based on the outcome of this pull request.  Then it can be copied and pasted to a #docs wiki page.